frontend/src/model/beta/Releases.tsx

```diff
@@ -26,21 +40,25 @@ export default function Releases({ model }: { model: any }) {
             Draft new Release
           </Button>
         </Box>
+        {isReleasesLoading && <Loading />}
+
+        {releases.length === 0 && <EmptyBlob text={`No releases found for model ${model.name}`} />}
 
-        {model.releases.map((release) => {
-          return (
-            release.name && (
-              <ModelReleaseDisplay
-                key={release.semver}
-                modelId={model.id}
-                release={release}
-                latestRelease={latestRelease}
-              />
+        {sortedReleases &&
+          sortedReleases.map((release) => {
```

Comment:
Again sorry for not noticing this the first time around, but this logic is effectively a `reduce` written as a `map`. I prefer to avoid this as if `release.name` was a falsy value that React will render (e.g. `0`) then you can end up with unintended elements being rendered on the page. That might not have any tangible effect in this case, but I believe it's a pattern we should avoid.

 I also always prefer to put this sort of logic in a `useMemo` so that we're not iterating through the array on every render. Consider changing to:
```
  const modelReleaseDisplays = useMemo(
    () =>
      sortedReleases.reduce<ReactElement[]>((releaseDisplays, release) => {
        if (release.name) {
          releaseDisplays.push(
            <ModelReleaseDisplay
              key={release.semver}
              modelId={model.id}
              release={release}
              latestRelease={latestRelease}
            />
          )
        }
        return releaseDisplays
      }, []),
    [latestRelease, model.id, sortedReleases]
  )
```